### ISSUE TITLE
Studio AppImage crashes when clicking "play" on the camera on Fedora 37

### ISSUE THREAD

**Issue by IOExceptional** [bambulab/BambuStudio#1665]
The app closes out with no error dialogue after I click "play" on the camera

1. Open Bambu Studio 1.5.0-hotfix1 OR 1.6.0
2. Click on "Device" tab
3. Select printer from the left menu dropdown
4. Click "Play" below the black box 
5. App crashes

**Expected behavior**
Instead of 5., it should display the output from the camera

**Desktop (please complete the following information):**
 - OS: Fedora 37
 - Bambu Studio 1.6.0-preview1 AND 1.5.0-hotfix1 (using the AppImages)

**Comment by lanewei120** [comment]
thanks for reporting this issue

please help to check the libjpeg.so's version on your system
for example, on my laptop, it is libjpeg.so.62
I can find /lib64/libjpeg.so.62 on my laptop

**Comment by lanewei120** [comment]
@IOExceptional 

could you help to check the libjpeg's version on your Fedora

**Comment by IOExceptional** [comment]
@lanewei120 I have `libjpeg.so.62.3.0`

![image](https://user-images.githubusercontent.com/1388264/234606911-bdd20b3c-184d-4d7f-9f23-c82f529d362f.png)

**Comment by lanewei120** [comment]
so it should be the issue mentioned in https://github.com/bambulab/BambuStudio/issues/1189
currently the libjpeg version used in bambustudio is jpeg8
it has conflicts with jpeg6 on your system

we will upload a fedora version using jpeg6 in the future versions

**Comment by lanewei120** [comment]
another solution is to upgrade libjpeg to jpeg8
however, as We don't have Fedora 37, or 38 at our side
we can not confirm how to do this

**Comment by lanewei120** [comment]
we have downgraded the jpeg version to jpeg62
could you try the newest image on Fedora?https://github.com/bambulab/BambuStudio/releases/download/v01.06.02.04/Bambu_Studio_linux_fedora_v01.06.02.04-20230427094209.AppImage

**Comment by Matthew-Beckett** [comment]
> we have downgraded the jpeg version to jpeg62 could you try the newest image on Fedora?https://github.com/bambulab/BambuStudio/releases/download/v01.06.02.04/Bambu_Studio_linux_fedora_v01.06.02.04-20230427094209.AppImage

I have tested this on Fedora 37 and it's now working properly for me.

**Comment by WayneManion** [comment]
I am running Arch. I am running the AUR package (built from source) from here: https://aur.archlinux.org/packages/bambustudio-git

The application doesn't crash when I click the play button to view the camera stream, but I can't see any video either. It just says:

Loading Failed [2]!

The gdb logs from running the built-from-source version of the app is here: https://pastebin.com/bhUK8JAh

**Comment by KOGJet** [comment]
> > we have downgraded the jpeg version to jpeg62 could you try the newest image on Fedora?https://github.com/bambulab/BambuStudio/releases/download/v01.06.02.04/Bambu_Studio_linux_fedora_v01.06.02.04-20230427094209.AppImage
> 
> I have tested this on Fedora 37 and it's now working properly for me.

Could you do this with the latest release on Ubuntu? Im new to this and hoping it could be my fix.

**Comment by lanewei120** [comment]
@KOGJet for ubuntu, please try the ubuntu image instead of the fedora image
and do you have issue with ubuntu image?

**Comment by KOGJet** [comment]
I tried both the fedora and Ubuntu and both either crash or show a error saying this device cannot handle more conversations.

**Comment by lanewei120** [comment]
maybe your issue is another one
how about the flathub image? https://flathub.org/apps/com.bambulab.BambuStudio

**Comment by KOGJet** [comment]
how do I install this into linux ? it didn't download as an app image but a ref pack

**Comment by KOGJet** [comment]
I get this error when running the script to download through flatpak[ ](error: Flatpak system operation ConfigureRemote not allowed for user)

**Comment by lanewei120** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/106916061/8c619f2b-551f-45da-a7a8-c3f32d3e59b7)

**Comment by KOGJet** [comment]
yes I tried to run those lines and got the [](error: Flatpak system operation ConfigureRemote not allowed for user)). How do i fix that?

**Comment by lanewei120** [comment]
let's report an issue here
https://github.com/flathub/com.bambulab.BambuStudio/issues
